<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QDialog" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QLabel" name="fileLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件/文件夹</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openFileBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeFileBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendFileBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="sendPgsBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="sStatusLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>360</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择要发送的文件！</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
